packages/client/src/AdditionalApi.ts

```diff
@@ -1,5 +1,21 @@
 import { LionwebResponse } from "@lionweb/server-shared"
 import { ClientResponse, RepositoryClient } from "./RepositoryClient.js"
+import {
+    BulkImport,
+    FBAttachPoint,
+    FBBulkImport,
+    FBContainment, FBMetaPointer,
+    FBNode,
+    FBProperty, FBReference, FBReferenceValue
+} from "@lionweb/server-additionalapi";
+import { Builder as FBBuilder } from 'flatbuffers';
```

Comment:
Cosmetic: inconsistent layout in imports (;s and whitespace).

Reply:
Done